ui/packages/plot/src/Plot.svelte

```diff
@@ -32,8 +33,29 @@
 
 	$: darkmode = theme == "dark";
 
-	$: if (value && value.type == "altair") {
-		spec = JSON.parse(value["plot"]);
+	$: plot = value?.plot;
```

Comment:
When I was fixing bokeh, I noticed that the DOM was updating more than once for each event trigger. This was actually causing multiple copies of the plot to show up. 

I figured it was because all the reactivity was based on `value` and `value` is an object so I thought there was some unintended mutation of that object. Unpacking `value` into it's component parts fixed the problem and there is only one DOM update every time the value changes from the backend as far as I can tell.